SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -151,8 +152,9 @@ function Get-SecretInfo
     $pattern = [WildcardPattern]::new($Filter)
     Invoke-lpass 'ls','-l' |
         Where-Object { 
-            $_ -match $lsLongOutput | Out-Null
-            $pattern.IsMatch($Matches[2])
+            $IsMatch = $_ -match $lsLongOutput -and $pattern.IsMatch($Matches[2])
+            if (-not $IsMatch ) { Write-Debug -Message "No match for: $_ `nThis record will be ignored." }
```

Comment:
wait... this isn't quite right. We want to `Write-Debug` if `$_ -match $lsLongOutput` doesn't match anything because that is unusual. This logic will log if something matches `lsLongOutput` but doesn't match the Filter which is a fine and expected behavior that we don't need to log.

Reply:
@TylerLeonhardt I revised the logic accordingly.